src/content/configuration/index.md

```diff
@@ -396,9 +383,23 @@ module.exports = {
 }
 ```
 
-T> Want to rapidly generate webpack configuration file for your project requirements with few clicks away.
+## Use custom config file
+
+If for some reason you want to use custom config file depending on certain situations you can change this via command line by using the `--config` flag.
```

Comment:
i understand why title was shortened, but in the text body suggest to change `config file` -> `configuration file`

Reply:
That's the thing, we've got both `config` and `configuration` all over the docs and sometimes it's hard to justify why one was picked over another one. I don't wanna seem too picky here but perhaps this can be brought to some common option? Also not sure if that's worth it

Reply:
@byzyk Yep, we have both, but we can start with this opportunity to change it to Configuration